src/renderer/components/+catalog/catalog.tsx

```diff
@@ -57,7 +57,7 @@ export class Catalog extends React.Component {
   }
 
   addToHotbar(item: CatalogEntityItem) {
-    const hotbar = hotbarStore.getByName("default"); // FIXME
+    const hotbar = hotbarStore.getActive();
 
     if (!hotbar) {
       return;
```

Comment:
If there is no hotbar we should tell the user to create one. (Or will this function only be displayed to the user if there are hotbars)

Reply:
Story for another PR... currently Lens will create a hotbar if none exists (and does not allow to remove last one).

Reply:
okay sounds good.